code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -734,9 +734,18 @@
 	taste_mult = 100
 	can_synth = FALSE
 
+/datum/reagent/consumable/nutriment/peptides
+	name = "Peptides"
+	color = "#BBD4D9"
+	taste_description = "umami"
+	description = "These restorative peptides not only speed up wound healing, but are nutrious as well!"
+	nutriment_factor = 10 * REAGENTS_METABOLISM // 33% less than nutriment to reduce weight gain
+	brute_heal = 3
+	burn_heal = 1
+
 /datum/reagent/consumable/caramel
-	name = "Caramel"
```

Comment:
what

Reply:
That line is still there when i view the file.